external-builds/pytorch/build_prod_wheels.py

```diff
@@ -360,15 +360,28 @@ def do_build(args: argparse.Namespace):
             "Please specify --pytorch-rocm-arch (e.g., gfx942)."
         )
 
+    rocprofiler_path = root_dir / "lib" / "librocprofiler64.so"
+    roctracer_path = root_dir / "lib" / "libroctracer64.so"
+
     env: dict[str, str] = {
         "CMAKE_PREFIX_PATH": str(cmake_prefix),
         "ROCM_HOME": str(root_dir),
         "ROCM_PATH": str(root_dir),
         "PYTORCH_ROCM_ARCH": pytorch_rocm_arch,
-        # TODO: Fix source dep on rocprofiler and enable.
-        "USE_KINETO": "OFF",
+        "USE_KINETO": os.environ.get("USE_KINETO", "ON"),
     }
 
+    # Only set Kineto-related CMake args if USE_KINETO is ON
+    if env["USE_KINETO"] == "ON":
+        env["PYTORCH_CMAKE_ARGS"] = (
+            f"-DKINETO_HIP_LIBRARY={rocprofiler_path}"
+            f"-DROCTRACER_LIBRARY={roctracer_path}"
+        )
+    else:
+        env["PYTORCH_CMAKE_ARGS"] = ""
+
+    print(f"  PYTORCH_CMAKE_ARGS = {env['PYTORCH_CMAKE_ARGS']}")
```

Comment:
I can't find references or documentation for this `PYTORCH_CMAKE_ARGS` environment variable. I also don't see it mentioned in the logs at https://github.com/ROCm/TheRock/actions/runs/16599403652/job/46954518344. Can you link me to how this is used?

Reply:
CI build is successful: https://github.com/ROCm/TheRock/actions/runs/16627391951/job/47047512082